git-annex/Singularity.git-annex.6.20180227

```diff
@@ -0,0 +1,25 @@
+Bootstrap: shub
+From:      Characterisation-Virtual-Laboratory/CharacterisationVL-Software:1804
+
+%labels
+    MAINTAINER_NAME  Jay van Schyndel
+    MAINTAINER_EMAIL jay.vanschyndel@monash.edu
+
+    APPLICATION_NAME ubuntu
```

Comment:
Do we need to changes this label to match git-annex?

Reply:
Updated the label.